lib/engine/game/g_1848/game.rb

```diff
@@ -172,17 +174,21 @@ class Game < Game::Base
           },
           {
             name: '3',
-            distance: 3,
+            distance: [{ 'nodes' => %w[city offboard], 'pay' => 3, 'visit' => 3 },
+                       { 'nodes' => ['town'], 'pay' => 99, 'visit' => 99 }],
             price: 200,
             rusts_on: '6',
             num: 5,
             variants: [
               { name: '3+', distance: 3, price: 230 },
```

Comment:
Did you forget the + variants or something you plan to do later?

Reply:
something i plan to do later. In fact I already did in my forked repo :-)

Reply:
The distance change came after a rule that i noticed, all the trains are unlimited plus trains, including the variant train.
The variant (+ train) just adds one free gauge change